amical/tests/test_extraction.py

```diff
@@ -66,13 +66,14 @@ def test_add_infos_header_commentary(commentary_infos):
     munch.munchify(commentary_infos)
 
 
-def test_commentary_infos_keep(commentary_infos, astropy_versions):
-    # Check that commentary cards are removed or kept depending on astropy version
-
-    astropy_version, working_version = astropy_versions
-
-    if Version(astropy_version) < Version(working_version):
-        assert "HISTORY" not in commentary_infos.hdr
+ASTROPY_VERSION = Version(astropy.__version__)
+@pytest.mark.skipif(ASTROPY_VERSION < Version("5.0rc"), reason="...")
```

Comment:
lol. I meant `"..."` to be a placeholder, please be less lazy than I :)

Reply:
Oops haha, I'll change that locally in a few minutes :)